src/lib/components/topBar/TopBar.svelte

```diff
@@ -224,16 +226,69 @@
 			icon={Error}
 			name="About"
 			on:click={() => {
-				console.log("About");
+				aboutVisible = true;
 			}}
 		/>
 	</TopBarButton>
 </div>
 
+{#if aboutVisible}
+	<div class="pageblocker">
+		<div class="aboutBox">
+			<div style="position: absolute; left: 2%;">
```

Comment:
Please use the standardized modal that is being merged here: https://github.com/ECDAR-AAU-SW-P5/Ecdar-GUI-Web/pull/111/files#diff-4fa9b2f0e22c217008f008d25d07f473d343030aca709b90d8ab2bfb57d24770

Any popup should use this modal to ensure consistency in the UI.